package.json

```diff
@@ -138,14 +138,15 @@
     "semver": "5.6.0",
     "shelljs": "0.8.2",
     "style-loader": "0.23.1",
-    "ts-loader": "2.3.7",
+    "ts-loader": "4.0.0",
     "tsickle": "0.33.0",
     "tslint": "5.11.0",
     "tslint-config-airbnb": "5.11.0",
     "tslint-config-prettier": "1.15.0",
     "tslint-plugin-prettier": "1.3.0",
     "typescript": "2.9.2",
     "webpack": "4.21.0",
+    "webpack-cli": "^3.1.2",
```

Comment:
does mixing Webpack cli 3 and Webpack 4 work?

Reply:
I would have expected a webpack-cli@4.x.x too but 3.1.2 is the [latest version](https://github.com/webpack/webpack-cli/releases/tag/v3.1.2).

So the big change in webpack@4 is that the cli part was [extracted as its own package](https://webpack.js.org/migrate/4/#cli). This is why we need to install it now.

AFAICT it builds the demo fine.